doc/source/changelog.rst

```diff
@@ -0,0 +1,87 @@
+.. _ref_release_notes:
+
+Release notes
+#############
+
+This document contains the release notes for the ``ansys-geometry-core`` project.
```

Comment:
Is there any reason we can't change "the ``ansys-geometry-core`` project." to "PyAnsys Geometry"?

Reply:
Of course - let me change that